profiles/sam/terraform/sam/cloudsql.tf

```diff
@@ -0,0 +1,25 @@
+resource "random_id" "user-password" {
+  byte_length   = 16
+}
+
+resource "random_id" "root-password" {
+  byte_length   = 16
+}
+
+# Cloud SQL database
+module "cloudsql" {
+  source        = "github.com/broadinstitute/terraform-shared.git//terraform-modules/cloudsql-postgres?ref=rl-postgres"
```

Comment:
Reminder to change to proper version tag before merging

Reply:
done